airbyte-integrations/connectors/source-trello/source_trello/spec.json

```diff
@@ -25,6 +25,15 @@
         "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}.[0-9]{3}Z$",
         "description": "UTC date and time in the format 2017-01-25T00:00:00Z. Any data before this date will not be replicated.",
         "examples": ["2021-03-01T00:00:00.000Z"]
+      },
+      "board_ids": {
+        "type": "array",
+        "items":{
+          "type": "string"
+        },
+        "title": "Trello board IDs",
+        "pattern": "^[a-zA-Z0-9,]*$",
```

Comment:
Hello,
some corrections and improvements about the pattern:

1. "pattern" field has to be inside the "items" section
2.  pattern has to validate one ID item ( not all string ) it means you don't need comma `,` in pattern.
you can use `^[a-z0-9]{24}$`, as I see we don't need upper case letters too.